plugin-flex-ts-template-v2/src/feature-library/send-audio-rec-file/custom-components/AudioRecorder/AudioRecorder.tsx

```diff
@@ -32,7 +33,11 @@ class AudioRecorder extends React.Component<Props, AudioRecorderState> {
     return (
       <div className="Twilio-MessageInputActions-default">
         <IconButton icon="Voice" onClick={this.openHideRecorder} />
-        <AudioRecorderPanel showRecorder={this.state.showRecorder} />
+        {this.state.showRecorder ? (
```

Comment:
I think this change caused the recorder to no longer show the last recording if I close the recorder and reopen it. This also results in the recorder incorrectly attempting to attach multiple recordings.

I'm guessing this was added relating to the `afterSendMessage` action listener. I think you could move that action listener into this file, that way you could restore this line to be like before to fix this issue.